packages/components/src/molecules/ReviewCard/ReviewCard.tsx

```diff
@@ -0,0 +1,146 @@
+import React, { forwardRef, useEffect, useRef, useState } from 'react'
+import type { HTMLAttributes } from 'react'
+import Rating from '../Rating'
+import Link from '../../atoms/Link'
+import ReviewAuthor from './ReviewAuthor'
+import ReviewDate from './ReviewDate'
+
+export interface ReviewCardProps extends HTMLAttributes<HTMLDivElement> {
```

Comment:
```suggestion
export interface ReviewCardProps extends HTMLAttributes<HTMLDivElement>, Partial<ReviewAuthorProps>, Partial<ReviewDateProps> {
```

Maybe we can use it this way to avoid listing all the props again. What do you think?

Reply:
Done ✅

Reply:
Question: All the ReviewAuthorProps and ReviewDateProps can be optional here?

Reply:
In the ReviewCard, it can.

I explain why i did this way here: 
https://github.com/vtex/faststore/pull/2645#discussion_r1960834802

Reply:
Nice! Then using Partial will do 👍